src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Utf8Formatter/FormattingHelpers.cs

```diff
@@ -149,32 +149,6 @@ public static void WriteTwoDecimalDigits(uint value, Span<byte> buffer, int star
 
         #endregion UTF-8 Helper methods
 
-        #region Math Helper methods
-
-        /// <summary>
-        /// We don't have access to Math.DivRem, so this is a copy of the implementation.
-        /// </summary>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static ulong DivMod(ulong numerator, ulong denominator, out ulong modulo)
-        {
-            ulong div = numerator / denominator;
-            modulo = numerator - (div * denominator);
-            return div;
-        }
-
-        /// <summary>
-        /// We don't have access to Math.DivRem, so this is a copy of the implementation.
```

Comment:
Another one with similar comment about this helper: https://github.com/dotnet/runtime/blob/ea35f6aae9e1aac0e0c5de6df97e05bd992a7528/src/libraries/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.Date.cs#L135-L138

Reply:
System.Text.Json is compiled downlevel, so it would need ifdefs

Reply:
True, maybe comment can be update since there is no System.Buffers.Text.FormattingHelpers.DivMod after this change.

Reply:
True, maybe comment can be updated since there is no System.Buffers.Text.FormattingHelpers.DivMod after this change.